index.html

```diff
@@ -15,9 +15,12 @@
   <body>
    <header class="header">
     <nav id="navegation"> 
-      <a class="header__menu__link" href="#About"> About</a></li>
-      <a class="header__menu__link" href="#Eperience">Experience</a></li>
-      <a class="header__menu__link" href="#Connect">Connect</a></li>
+      <a class="header__menu__link" href="#About"> About</a>
+      <a class="header__menu__link" href="#Experience">Experience</a>
+      <a class="header__menu__link" href="#Connect">Connect</a>
+      <a class="header__menu__link" href="#Skills">Skills</a>
+      <a class="header__menu__link" href="#Projects">Projects</a>
```

Comment:
You didn't use capital letters for the skills and projects sections. The id is case sensitive so you need to update your sections to use `Skills` and `Projects` as their ids. Or you can update the links in your `nav` to use the current ids, which are `skills` and `projects`.

Reply:
I suggest keeping the first letters capitalized to follow the same convention as the other sections.